dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/EventOperationParamsMapper.java

```diff
@@ -253,28 +251,25 @@ private void validateAttributeOptionCombo(CategoryOptionCombo attributeOptionCom
     }
   }
 
-  private void validateOrgUnitMode(EventOperationParams params, User user, Program program)
+  private void validateOrgUnitMode(
+      OrganisationUnitSelectionMode orgUnitMode, User user, Program program)
       throws BadRequestException {
-    if (params.getOrgUnitMode() != null) {
-      String violation = getOrgUnitModeViolation(params, user, program);
-      if (violation != null) {
-        throw new BadRequestException(violation);
-      }
+
+    String violation = getOrgUnitModeViolation(orgUnitMode, user, program);
```

Comment:
Can we inline this method `getOrgUnitModeViolation`? I think its only used once

Reply:
It's true it's just used once, I just followed this mapper's pattern, above there are the methods `validateProgram`, `validateProgramStage` and `validateRequestedOrgUnit`, all of them also just used once.

Reply:
I mean inline method `getOrgUnitModeViolation`. You would still have `validateOrgUnitMode`